packages/grafana-ui/src/components/uPlot/types.ts

```diff
@@ -3,7 +3,7 @@ import uPlot, { Options, Series, Hooks } from 'uplot';
 import { DataFrame, TimeRange, TimeZone } from '@grafana/data';
 import { UPlotConfigBuilder } from './config/UPlotConfigBuilder';
 
-export type PlotSeriesConfig = Pick<Options, 'series' | 'scales' | 'axes'>;
+export type PlotSeriesConfig = Pick<Options, 'series' | 'scales' | 'axes' | 'cursor'>;
```

Comment:
@dprokop -- should this be renamed `PlotConfig` -- it will eventually be everything the `UPlotConfigBuilder` knows how to build

Reply:
Yes, we can rename it!